### DISCUSSION TITLE
Unexpected zombie task in Cloud Composer Environment

### DISCUSSION THREAD

**Discussion by RESArnau** [apache/airflow#36556] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### If "Other Airflow 2 version" selected, which one?

2.5.3

### What happened?

In our Google Cloud Composer environment, we have been experiencing a series of unexpected task failures in result of a task mark as zombie. These failures do not appear to correlate with system resource constraints, as both CPU and memory usage are below their respective limits according to our monitoring dashboards. The task is just waiting for a message in a pub/sub queue. It only happens in one of our two enviroments. 

### What you think should happen instead?

Tasks should keep beeing executed without being marked as zombie when system resources are underutilized, and no other apparent issues are present that would cause such failures.

### How to reproduce

The issue can be sporadically observed in our Cloud Composer environment when executing tasks. It is not consistently reproducible but occurs frequently enough to impact our workflows.

### Operating System

composer-2.5.1

### Deployment

Google Cloud Composer

### Deployment details

composer-2.5.1

Scheduler:
2 schedulers with 1.5 vCPU, 3 GB memory, 1 GB storage each

Triggerer:
1 triggerer with 1 vCPU, 1 GB memory, 1 GB storage

Web server:
1 vCPU, 3 GB memory, 1 GB storage

Worker:
Autoscaling between 1 and 6 workers, with 1.5 vCPU, 3 GB memory, 1 GB storage each

Environment size:
Small

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Without more details - there is no way this can be acted on. If you can provide more details, logs and finding - then ther is a higher chance (no guarantees though) that someone might take a look. Other than that - you can raise the issue to composer team - unlike here, composer team has paid support and you can - I believe - give them permissions to look at your logs and maybe they can provide more information. Even if ti is an airflow issue - people here need more evidences and details from the environment that could be helpful with diagnosis.


Converting that into a discussion - we can always convert it back to issue if more details are provided - either by you or composer support investigating it.
